src/main/java/com/socialmedia/controller/ChatMessageController.java

```diff
@@ -28,8 +31,12 @@ public ChatMessageController(ChatMessageMapper chatMessageMapper) {
   @GetMapping("/{chatId}")
   public ResponseEntity<List<ChatMessageDtoOut>> getAllMessagesForChat(
             @PathVariable Long chatId,
-            @PageableDefault(sort = { "id" }, direction = Sort.Direction.DESC) Pageable pageable) {
+            @PageableDefault(sort = { "id" }, direction = Sort.Direction.ASC) Pageable pageable) {
```

Comment:
Are you sure about ascending? This is pageable, so we'll always receive the very first messages of our chat, when asking for first page. But in chat you'd want to see the last ones

Reply:
I changed it cause I render messages from top to bottom. But I didn't reckon that I receiving only last 10. I`ll find out how to manage it.